slam_toolbox/lib/karto_sdk/include/karto_sdk/contrib/ChowLiuTreeApprox.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2022 Ekumen Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
```

Comment:
This project is LGPL2.1, any contribution made should be made under these terms

Reply:
Ah, I replicated SRI code license. We can certainly transition to LGPL v2.1. I'll review the terms first.

Reply:
Changed in https://github.com/SteveMacenski/slam_toolbox/pull/468/commits/973f3b70dc16479107d51a5442aa08b58bd743eb.